src/main/java/org/opensearch/flowframework/rest/RestDeleteWorkflowAction.java

```diff
@@ -62,6 +63,7 @@ public List<Route> routes() {
     @Override
     protected BaseRestHandler.RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
         String workflowId = request.param(WORKFLOW_ID);
+        request.param(CLEAR_STATUS); // consume and ignore, we will pass params to workflow
```

Comment:
Just to refresh my memory, why do we need to invoke `param()` for each of these? We consume all the params either way when we send a transport request

Reply:
First, I had neglected to push a commit I'd saved locally that changes this to actually pull just this param and send just that map over transport.  About to do that now.

But the answer to the larger question is that if we throw an exception _before_ getting to the `params()` call later, such as if the plugin is disabled, we won't consume it and will get a misleading error message.

When I first tried to locally run integ tests before putting this here, I got this:

> > Task :integTest
> 
> REPRODUCE WITH: ./gradlew ':integTest' --tests "org.opensearch.flowframework.rest.FlowFrameworkRestApiIT.testCreateAndProvisionRemoteModelWorkflow" -Dtests.seed=FBBABD239888A9F8 -Dtests.security.manager=false -Dtests.locale=en-AU -Dtests.timezone=Antarctica/Macquarie -Druntime.java=17
> 
> org.opensearch.flowframework.rest.FlowFrameworkRestApiIT > testCreateAndProvisionRemoteModelWorkflow FAILED
>     org.opensearch.client.ResponseException: method [DELETE], host [http://127.0.0.1:56350], URI [/_plugins/_flow_framework/workflow/qai-xY8B6INOlEYKiSbi?clear_status=true], status line [HTTP/1.1 400 Bad Request]
>     {"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"request [/_plugins/_flow_framework/workflow/qai-xY8B6INOlEYKiSbi] contains unrecognized parameter: [clear_status]"}],"type":"illegal_argument_exception","reason":"request [/_plugins/_flow_framework/workflow/qai-xY8B6INOlEYKiSbi] contains unrecognized parameter: [clear_status]"},"status":400}

Reply:
Similar to https://github.com/opensearch-project/flow-framework/pull/295?

Reply:
> Similar to #295?

Exactly.